artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -107,6 +108,12 @@
    // Default address drop threshold, applied to address settings with BLOCK policy.  -1 means no threshold enabled.
    public static final long DEFAULT_ADDRESS_REJECT_THRESHOLD = -1;
 
+   public static final DeadLetterAddressRoutingType DEFAULT_DEAD_LETTER_ADDRESS_PREFIX_ROUTING_TYPE = DeadLetterAddressRoutingType.MULTICAST;
+
+   public static final boolean DEFAULT_DEAD_LETTER_ADDRESS_PREFIX_DURABLE = true;
```

Comment:
Default probably should match the parent queue.

Reply:
Just to confirm, for all 3 settings?

Reply:
Found another comment, which I understand says all 3 should be taken from parent if null.
Closing this one then.